### ISSUE TITLE
Negating config values when using config()

### ISSUE THREAD

**Issue by klickreflex** [tailwindcss/discuss#126]
I'm trying to use a margin value from Tailwind's config using `config('margin.3')` and need to prefix that with a minus to negate the value. I tried:

-  using`config('negativeMargin.3')`: does not help as the value itself isn't negative
- `- config('margin.3)` / `-config('margin.3)`: config() doen't get interpreted
- `config(-'margin.3))`: result is undefined

The only solution I found working is: `calc (-1 * config('margin.3'))`. Obviously that is one hell of an unnecessary calculation, so really don't want to go down that path. Any other ideas?

**Comment by adamwathan** [comment]
Good catch, hadn't thought of this. I'm not sure what the best fix is but it will definitely involve a change to Tailwind itself. Guessing for your situation you can't just use `@apply .-m-3;` or similar?

**Comment by klickreflex** [comment]
No unfortunately not, I'm using it in a box-shadow context (where it also makes no sense to put the shadow):  

```
    // Tailwind variables should be used instead of .7rem, but it's not possible
    // to negate these, see https://github.com/tailwindcss/discuss/issues/126
    box-shadow: .7rem 0 0 config('colors.blue'), -.7rem 0 0 config('colors.blue');
```

**Comment by donnysim** [comment]
Trying to achieve this for an hour now and nothing works, and it's basically how postcss works I guess?

I tried every way possible, cheating up functions, variables etc and it's just impossible to convert the `config` function result into negative value. What I tried:

```
right: -config('sidebar.width');

right: -#{config('sidebar.width')};

$tmp: config('sidebar.width');
right: -$tmp;
right: -#{$tmp};
```

and a lot more, and it just compiles into `-config("sidebar.width")` css value.

Adding 
```js
'-config': (path, defaultValue) => {
  return '-' + _lodash2.default.get(config(), _lodash2.default.trim(path, `'"`), defaultValue);
}
```
to `evaluateTailwindFunctions.js` solves it :\

**Comment by jakewtaylor** [comment]
Another work around for now is to just add extra keys to your `tailwind.js`, for example to create a negative version of `width.6`:

```JavaScript
module.exports = {
  // ...
  width: {
    // ...
    'negative-6': '-1.5rem',
    // You can add as many as you need here
  },
  // ...
}
```

And then use them like before:

```JavaScript
config('width.negative-6');
```
